backend/model/movies-model.js

```diff
@@ -0,0 +1,13 @@
+// Load mongoose package
+import mongoose from 'mongoose';
+
+// Create a schema
+const MovieSchema = new mongoose.Schema({
```

Comment:
Wondering if there's any case where we need just the schema without the mongoose, don't know what's best practice, to keep it all in the same file or separate.